src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -682,11 +726,14 @@ fx_definition_t* fx_muxer_t::resolve_fx(
 
             if (pal::directory_exists(fx_dir))
             {
-                //Continue to search for a better match if available
-                std::vector<fx_ver_t> version_list;
-                version_list.push_back(resolved_ver);
-                version_list.push_back(selected_ver);
-                resolved_ver = resolve_framework_version(version_list, fx_ver, specified, config.get_patch_roll_fwd(), config.get_roll_fwd_on_no_candidate_fx());
+                if (selected_ver != fx_ver_t(-1, -1, -1))
```

Comment:
Review note: this avoids doing an unnecessary comparison against (-1, -1, -1) for the first loop